### PR TITLE
Create dummy files and components for add camp flow

### PR DESCRIPTION
**Pull request by e-wai**
## Implementation description
* Very basic setup work for add camp

## Steps to test
1. Go to `/admin/camp/add-camp` and enjoy
2. Buttons should toggle components

## What should reviewers focus on?
* See comments

https://user-images.githubusercontent.com/40974372/198850524-af882bd3-971e-460d-8025-838d6c19d631.mov

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/pages/CampCreation/ScheduleSessions/index.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Box, Text } from "@chakra-ui/react";
+
+const ScheduleSessionsPage = (): JSX.Element => {
```
Comment (e-wai) [inline frontend/src/components/pages/CampCreation/ScheduleSessions/index.tsx:4]: Preference for `JSX.Element` vs. `React.ReactElement`?

File: frontend/src/components/pages/CampCreation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      <Button
+        onClick={() => setCurrentPage(CampCreationPages.ScheduleSessionsPage)}
+      >
+        Schedule Sessions
+      </Button>
+      <Button
+        onClick={() => setCurrentPage(CampCreationPages.RegistrationFormPage)}
+      >
+        Registration Form
+      </Button>
```
Comment (e-wai) [inline frontend/src/components/pages/CampCreation/index.tsx:46]: Eventually should be its own component

File: frontend/src/components/pages/CampCreation/ScheduleSessions/index.tsx (hunk 1 above)
Comment (twilkhoo) [reply frontend/src/components/pages/CampCreation/ScheduleSessions/index.tsx:4]: Looks like we use both in the codebase haha... but I prefer JSX.Element like you've done.

File: frontend/src/components/pages/CampCreation/CampDetails/index.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Box, Text } from "@chakra-ui/react";
+
+const CampDetails = (): JSX.Element => {
```
[the remark this answers is not in the document]
Comment (twilkhoo) [reply frontend/src/components/pages/CampCreation/CampDetails/index.tsx:4]: CampCreationDetails?

**Review by twilkhoo — APPROVED** [review]
LGTM, this is a good base for the stepper component!

File: frontend/src/components/pages/CampCreation/index.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        return <ScheduleSessions />;
+      case CampCreationPages.RegistrationFormPage:
+        return <RegistrationForm />;
+      default:
+        throw new Error("never reached");
+    }
+  };
+
+  return (
+    <Box>
```
Comment (twilkhoo) [inline frontend/src/components/pages/CampCreation/index.tsx:33]: You could perhaps consider adding some margins here, but if it's unneccesary for a dummy component that's fine too.

**Review by twilkhoo — APPROVED** [review]

File: frontend/src/components/pages/CampsPage.tsx [hunk 5]
```diff
@@ -1,10 +1,12 @@
 import React from "react";
+import { Box, Button, Text } from "@chakra-ui/react";
```
Comment (twilkhoo) [inline frontend/src/components/pages/CampsPage.tsx:2]: I think you only need `Box` here

**Review by dhmi19 — APPROVED** [review]

**Pull request merged**